crowbar_framework/app/models/pacemaker_service.rb

```diff
@@ -33,14 +33,14 @@ def role_constraints
           "unique" => false,
           "count" => 32,
           "platform" => {
-            "suse" => "/11\.[3-9]/"
+            "suse" => "/.*/"
```

Comment:
Yes, the cookbooks are not fully ready for non-SUSE (and this was actually never tested), so we make it impossible to use that in the webui for now.